types/noble/index.d.ts

```diff
@@ -9,9 +12,13 @@
 import events = require("events");
 
 export declare function startScanning(): void;
+export declare function startScanning(callback: Function): void;
```

Comment:
Not exactly sure what you mean here. Would this be better?

`export declare function startScanning(callback?: (error?: Error) => void): void;`